### PR TITLE
Updates for 0.11

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -16,11 +16,14 @@
...
+    "purescript-dom": "^4.0.0",
+    "purescript-aff": "^3.0.0",
+    "purescript-arraybuffer-types": "^2.0.0"
   },
   "devDependencies": {
-    "purescript-console": "^2.0.0"
+    "purescript-console": "^3.0.0"
+  },
+  "resolutions": {
+    "purescript-arraybuffer-types": "^2.0.0"
```
Comment (nwolverson) [inline bower.json]: Is this actually required? I wouldn't expect to publish a library with any resolutions section

**Review by nwolverson — COMMENTED** [review]
Thanks. Will take a look tonight

File: bower.json (hunk 1 above)
Comment (jacereda) [reply bower.json]: Yeah, maybe it's better to wait, if this gets merged it won't be necessary:

https://github.com/purescript-contrib/purescript-dom/pull/96/commits/28cbd24757301db60d966279667bbaf13c92b338

**Comment by nwolverson** [comment]
@jacereda is that referenced commit eventually merged? it looks to me like you closed your PR without incorporating the changes elsewhere, and the arraybuffer-types is still the older version in purescript-dom

**Comment by jacereda** [comment]
Only the documentation changes in that PR were actually merged, and looks like almost everything is using pre-2.0 -arraybuffer-types.

**Pull request merged**